web/dataselectie/datasets/brk/documents.py

```diff
@@ -322,13 +324,13 @@ def doc_from_eigendom(eigendom: object) -> Eigendom:
     else:
         doc.ggw_code = ['']
         doc.ggw_naam = ['']
-    wijken = kot.wijken.all()
-    if wijken:
-        doc.wijk_naam = [wijk.naam for wijk in wijken]
-        doc.wijk_code = [wijk.code for wijk in wijken]
+    buurtcombinaties = kot.wijken.all()
+    if buurtcombinaties:
+        doc.buurtcombinatie_naam = doc.wijk_naam = [buurtcombinatie.naam for buurtcombinatie in buurtcombinaties]
```

Comment:
doc.wijk_naam en doc.wijk_code kan weg, toch?

Reply:
Het idee was dat het allebei nog werkte, maar als dat niet nodig is kan ik het weghalen